service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -173,4 +203,24 @@ private String careOrderDetails(int numOfChildren, String caseLocalAuthority) {
                     .formatLocalDateToString(child.getDateOfBirth(), FormatStyle.LONG) : ""))
             .collect(toList());
     }
+
+    private String getChildrenDescription(EPOChildren epoChildren) {
+        if (epoChildren.getDescriptionNeeded().equals("Yes")) {
+            return epoChildren.getDescription();
+        }
+
+        return "";
+    }
+
+    private String formatEPOEndDateTime(LocalDateTime dateTime) {
+        return dateFormatterService.formatLocalDateTimeBaseUsingFormat(dateTime, "d MMMM yyyy 'at' h:mma");
```

Comment:
Not something to worry about now but at some point we should probably refactor all of these format strings into constants

Reply:
Yes good idea, have you a ticket raised?

Reply:
Not yet, just had a talk with Adam about it